doc/source/whatsnew/v0.18.1.txt

```diff
@@ -21,6 +21,47 @@ Highlights include:
 New features
 ~~~~~~~~~~~~
 
+.. _whatsnew_0181.enhancements.groubynth:
+
+Index in ``Groupby.nth`` output is now more consistent with ``as_index``
+argument passed in (:issue:`11039`):
+
+Previous Behavior:
+
+.. code-block:: ipython
+
+    In [4]: df
+    Out[4]:
+       A  B
+    0  a  1
+    1  b  2
+    2  a  3
+
+    In [5]: df.groupby('A', as_index=True)['B'].nth(0)
+    Out[5]:
+    0    1
+    1    2
+    Name: B, dtype: int64
+
+
+New Behavior:
+
+.. code-block:: ipython
```

Comment:
@jreback from what i see from similar cases, this is already in correct form; [v0.18.1.txt#L159](https://github.com/pydata/pandas/blob/bac68d6bb9ea22e9e744fe709d1270f257d71e0f/doc/source/whatsnew/v0.18.1.txt#L159) as an example